var/spack/repos/builtin/packages/flux/package.py

```diff
@@ -26,12 +26,13 @@
 import os
 
 
-class Flux(Package):
+class Flux(AutotoolsPackage):
     """ A next-generation resource manager (pre-alpha) """
 
     homepage = "https://github.com/flux-framework/flux-core"
-    url      = "https://github.com/flux-framework/flux-core"
+    url      = "https://github.com/flux-framework/flux-core/releases/download/v0.6.0/flux-core-0.6.0.tar.gz"
 
+    version('0.6.0', md5='d44a0f719744771d168edd205bd8e74e')
```

Comment:
Didn't actually know the second argument was called md5. Can you leave out the `md5=` part just to match other packages?